src/hotspot/cpu/aarch64/aarch64.ad

```diff
@@ -16935,6 +16941,80 @@ instruct encode_ascii_array(iRegP_R2 src, iRegP_R1 dst, iRegI_R3 len,
   ins_pipe(pipe_class_memory);
 %}
 
+//----------------------------- CompressBits/ExpandBits ------------------------
+
+instruct compressBitsI_reg(iRegINoSp dst, iRegIorL2I src, iRegIorL2I mask,
+                           vRegF tdst, vRegF tsrc, vRegF tmask) %{
+  match(Set dst (CompressBits src mask));
+  effect(TEMP tdst, TEMP tsrc, TEMP tmask);
+  format %{ "mov    $tsrc, $src\n\t"
+            "mov    $tmask, $mask\n\t"
+            "bext   $tdst, $tsrc, $tmask\t# parallel bit extract\n\t"
```

Comment:
We don't need to tell readers what BEXT does.;

Reply:
I can remove that - it is just to match the equivalent x86 code. My actual preference would be to emit the method the intrinsic was replacing.